PilotDataPlatform/search
# PILOT-1533 Search items by attributes

## Summary

- Allow searching ElasticSearch `metadata-items` by attribute values.
	- Attributes parameter accepts a JSON object (e.g. `{"key1":["value1","value2","value3"],"key2":"value1"}`).
	- Lists are interpreted as contains / match any. Strings are interpreted as must match exactly.
- Uses the new attributes format from: https://github.com/PilotDataPlatform/metadata-event-handler/pull/13

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1533

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

Copy and paste attributes JSON objects from Portal API calls and see if they are handled correctly.

<img width="1167" src="https://user-images.githubusercontent.com/60715210/186511532-130872a5-a157-49bf-a770-004f6757639d.png">
